RFC.rst

```diff
@@ -62,8 +62,13 @@ to fairly and consistently applying these principles to every aspect of
 managing this project. Project maintainers who do not follow or enforce
 the Code of Conduct may be permanently removed from the project team.
 
-This Code of Conduct applies both within project spaces and in public
-spaces when an individual is representing the project or its community.
+This Code of Conduct applies in all project spaces, as well as in other
+spaces where a user is acting in an official role as a spokesperson for
+the project.
+
+It does not apply to speech which does not pertain to be on behalf of the
+project (for example, opinions shared on social media which are not
+posted on an official project account).
```

Comment:
I like how [the Go Code](https://golang.org/conduct#spaces) addresses that question:

- Explicit enforcement in official project spaces, which are explicitly listed: the projects and code reviews themselves, the mailing lists, the IRC channel, and the subreddit.
- Participants and contributors are "encouraged to follow the Code" any time they are participating or contributing.

My read of the second point is that if a Go contributor is exhibiting any of the [unwelcome behavior](https://golang.org/conduct#unwelcome_behavior) on e.g. Twitter, the project might suggest that the behavior is against their core values, but would not take any project-level action.

I like the clear distinction they make between "Go spaces" and "non-Go spaces", because it a) sets clear expectations for the community, and b) doesn't bite off more than an OSS project community can realistically chew (in terms of policing behavior across the internet).